web3-react/.eslintrc.json

```diff
@@ -0,0 +1,62 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "root": true,
+    "extends": [
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react-hooks/recommended",
+        "plugin:prettier/recommended"
```

Comment:
Try extending @uniswap/eslint-config present instead of a custom one!

Reply:
I just pulled this from our previous examples, but update it here!